src/main/flight/pid.c

```diff
@@ -344,6 +344,12 @@ static void pidApplyRateController(const pidProfile_t *pidProfile, pidState_t *p
         pidState->errorGyroIfLimit = ABS(pidState->errorGyroIf);
     }
 
+#ifdef USE_SERVOS
```

Comment:
I think `USE_SERVOS` is redundant here. Checking for `FIXED_WING` state should probably be enough?